### PR TITLE
Fixing incorrect icon shown  issue#1316

### LINKED ISSUE #1316: Incorrect icon shown
**Summary:**

When user tries to open user profile in absence of internet connection, the message shown contains incorrect icon.

**Steps to reproduce:**

* Login into the app
* Turn off mobile data/Wifi
* Open user profile 
* See the screen being displayed

**Expected behaviour:**

The icon should be replaced with an appropriate one

**Screenshots:**
WhatsApp Image 2020-02-05 at 4 50 05 PM

### PR DESCRIPTION
**Pull request by som23ya**
replacement with assignment_not_turned icon

Fixes #1316 

Screenshot (138)

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ buildscript {
         maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.2'
+        classpath 'com.android.tools.build:gradle:3.5.0'
```
Comment (devansh-299) [inline build.gradle]: this change is not required please remove this from commit

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Tue Oct 10 11:28:53 IST 2017
+#Mon Feb 10 02:05:55 IST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```
Comment (devansh-299) [inline gradle/wrapper/gradle-wrapper.properties]: Same goes for this file

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#Tue Oct 10 11:28:53 IST 2017
+#Mon Feb 10 02:05:55 IST 2020
```
Comment (devansh-299) [inline gradle/wrapper/gradle-wrapper.properties]: please remove this change as it is unnecessary and squash all your commits
rest everything seems fine

**Review by devansh-299 — CHANGES_REQUESTED** [review]

**Review by som23ya — COMMENTED** [review]
@devansh-299 thanks for the guidance. :D please review this.

**Review by devansh-299 — APPROVED** [review]

**Comment by devansh-299** [comment]
The changes are fine but please squash your commits

**Review by ShivangiSingh17 — APPROVED** [review]

**Comment by luckyman20** [comment]
Issue has already been resolved.

**Pull request closed without merging**